packages/f-checkout/src/components/Checkout.vue

```diff
@@ -35,7 +35,6 @@
                         </error-message>
                     </template>
                 </form-field>
-
```

Comment:
Not one to change now necessarily, but it's best to keep whitespace between elements like this as it makes the file easier to scan - especially as components get larger it can be hard to pick out where sibling elements sit alongside each other.